src/test/java/it/gov/pagopa/pu/registry/mapper/debtposition/DebtPositionRegistryMapperServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package it.gov.pagopa.pu.registry.mapper.debtposition;
+
+import it.gov.pagopa.pu.registry.event.payments.dto.*;
+import it.gov.pagopa.pu.workflowhub.dto.generated.DebtPositionDTO;
+import it.gov.pagopa.pu.workflowhub.dto.generated.PaymentEventType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class DebtPositionRegistryMapperServiceTest {
+
+  private DebtPositionRegistryMapperService debtPositionRegistryMapperService;
+  private DebtPositionEventDTO2DebtPositionRegistryMapper debtPositionEventDTO2DebtPositionRegistryMapper;
+  private DebtPositionIoEventDTO2DebtPositionRegistryMapper debtPositionIoEventDTO2DebtPositionRegistryMapper;
+  private DebtPositionSendEventDTO2DebtPositionRegistryMapper debtPositionSendEventDTO2DebtPositionRegistryMapper;
+
+  @BeforeEach
+  void setUp() {
+    this.debtPositionEventDTO2DebtPositionRegistryMapper = Mockito.spy(new DebtPositionEventDTO2DebtPositionRegistryMapper());
+    this.debtPositionIoEventDTO2DebtPositionRegistryMapper = Mockito.spy(new DebtPositionIoEventDTO2DebtPositionRegistryMapper());
+    this.debtPositionSendEventDTO2DebtPositionRegistryMapper = Mockito.spy(new DebtPositionSendEventDTO2DebtPositionRegistryMapper());
+
+    debtPositionRegistryMapperService = new DebtPositionRegistryMapperService(
+        debtPositionEventDTO2DebtPositionRegistryMapper,
+        debtPositionIoEventDTO2DebtPositionRegistryMapper,
+        debtPositionSendEventDTO2DebtPositionRegistryMapper
+    );
+  }
+
+  @Test
+  void map_shouldReturnDebtPositionRegistry_whenEventIsDebtPositionEvent() {
```

Comment:
the test naming convention, as documented on the handbook, is different
https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/962232530/HB+Dev+-+Spring+Boot+Unit+Testing+Best+Practice#%40Test

```suggestion
  void givenDebtPositionEventWhenMapThenInvokeDebtPositionMapperService() {
```